lagunita/lms/templates/static_templates/tos.html

```diff
@@ -522,6 +524,13 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
                 date=effective_date
             ) | n }
         </strong></p>
+
+        <p>
+        ${_("""Please visit {copyright_link}Copyright Complaints{link_end} for information on how to submit a complaint.""").format(
```

Comment:
@glewis17 Is this the text requested on the ticket?

Reply:
@stvstnfrd The text as requested on the ticket doesn't make sense for Copyright Complaints.
@mondiaz Are you ok with this text?